src/util.f90

```diff
@@ -820,7 +820,7 @@ subroutine generate_bunch_bigaussian_weighted( &
  real(dp), allocatable :: ppcb_positions(:,:)
 
  bch=weight
- sigma_cut=3.0
+ sigma_cut=4.0
```

Comment:
Sounds appropriate, should we do that? At the end 4 is the best choice, we can now leave like this.